source/extensions/filters/network/postgres_proxy/postgres_message.h

```diff
@@ -11,7 +11,7 @@ namespace PostgresProxy {
 
 /**
  * Postgres messages are described in official Postgres documentation:
- * https://www.postgresql.org/docs/12/protocol-message-formats.html
+ * https://www.postgresql.org/docs/current/protocol-message-formats.html
```

Comment:
This will change over time? I'm OK with this though.

Reply:
The work we did here with the filter is to be totally independent of the Postgres version, so make sense to point to the current stable version.